numpy/core/src/multiarray/multiarraymodule.c

```diff
@@ -129,12 +129,18 @@ PyArray_GetPriority(PyObject *obj, double default_)
     ret = PyArray_LookupSpecial_OnInstance(obj, "__array_priority__");
     if (ret == NULL) {
         if (PyErr_Occurred()) {
-            PyErr_Clear(); /* TODO[gh-14801]: propagate crashes during attribute access? */
+            /* TODO[gh-14801]: propagate crashes during attribute access? */
+            PyErr_Clear();
         }
```

Comment:
Strange, this error condition is still not hit. Maybe the test needs a `__getitem__` that raises?

Reply:
ohhh, I did not modify this one, so didn't add a test for it.  Let me just add it, since it fits into the same "theme".